### PR TITLE
Feature/1 initial content

### LINKED ISSUE #2: Add initial content

### PR DESCRIPTION
**Pull request by nhinze23**
Resolves #2

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# k8s-longhorn Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+
```
Comment (robertauer) [inline CHANGELOG.md:8]: This empty line doesn't comply to keepachangelog.com
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+### Added
+
```
Comment (robertauer) [inline CHANGELOG.md:10]:
```suggestion
```

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+##@ Release
+
+.PHONY: generate-release-resource
+generate-release-resource: $(K8S_RESOURCE_TEMP_FOLDER)
+	@cp manifests/longhorn.yaml ${K8S_RESOURCE_TEMP_YAML}
+
+
+.PHONY: etcd-release
+setup-release: ## Interactively starts the release workflow.
```
Comment (robertauer) [inline Makefile:17]: - These targets don't have the same name
- Why aren't they called "longhorn-release"?

Comment (nhinze23) [reply Makefile:17]: CopyPasta.. :smile:

**Review by robertauer — APPROVED** [review]

**Pull request merged**